packages/common/src/hardhat/tasks/artifacts.ts

```diff
@@ -245,6 +245,31 @@ export async function getAddress(name: DeploymentName | ContractName, chainId: n
 }
 `
     );
+
+    // Write abi and bytecode for the browser file.
```

Comment:
might want to update this comment to reference matching the frontend api

Reply:
Yep, will fix!